.github/workflows/ci.yaml

```diff
@@ -18,16 +18,18 @@ jobs:
       matrix:
         env:
           - IMAGE: rolling-ci
-            CCOV: true
+            CCOV: false # Disabled: https://github.com/moveit/moveit2/issues/2866
```

Comment:
Please comment out instead of setting `CCOV: false`. As the image is identical to the next job, there is no benefit in running the job here as well.

Reply:
Hm, don't you think there is an advantage testing gcc separately from clang+clang-tidy? I assume that most users run exactly this setup, so I would prefer keeping it in.

Reply:
Ok. Sure.